doc/examples/reconst_cti.py

```diff
@@ -0,0 +1,172 @@
+and structural disorder in compartments like cross-sectional variance
+[NovelloL2022]_ [AlvesR2022]_.
+
+Although the CorrelationTensorModel and the DiffusionKurtosisTensorFit may
+share some similarities, they have significantly different representations for
+the diffusion-weighted signal. This difference leads to the necessity for a
+unique ``design matrix`` specifically for CTI.
+The CorrelationTensorModel expresses the diffusion-weighted signal as:
+
+.. math::
+    \begin{align}
+    \\log E_{\\Delta}(q_1, q_2) &= \\left(q_{1i}q_{1j}
+    + q_{2i}q_{2j}\right) \\Delta D_{ij} \\
+    &+ q_{1i}q_{2j}Q_{ij} \\
+    &+ \frac{1}{6} \\left( q_{1i}q_{1j}q_{1k}q_{1l}
+    + q_{2i}q_{2j}q_{2k}q_{2l} \right) \\
+    &\\quad \\times \\Delta^2 D^2 W_{ijkl} \\
+    &+ \frac{1}{4} q_{1i}q_{1j}q_{2k}q_{2l}Z_{ijkl} \\
+    &+ \\frac{1}{6} \\left( q_{1i}q_{1j}q_{1k}q_{2l}
+    + q_{2i}q_{2j}q_{2k}q_{1l} \right) S_{ijkl} \\
+    &+ O(q^6)
+    \\end{align}
+
+    where: $\\Delta D_{ij}$ refers to the elements of the total diffusion
+    tensor (DT) and $W_{ijkl}$ are the elements of the total kurtosis tensor
+    (KT) and $D$ is the mean diffusivity and $Q_{ij}$ are the elements of a 2nd
+    order correlation tensor Q.
+    $Z_{ijkl}$ and $S_{ijkl}$ are the elements of the 4th order displacement
+    correlation tensors Z and S, respectively.
+
+However it differentiates from the DiffusionKurtosis Model by calculating the
+different sources of kurtosis.
+
+In the following example we show how to fit the correlation tensor model on a
+real life dataset and how to estimate correlation tensor based statistics.
+
+First, we'll import all relevant modules.
+"""
+import matplotlib.pyplot as plt
+
+import dipy.reconst.cti as cti
+from dipy.io import read_bvals_bvecs
+from dipy.core.gradients import gradient_table
+from dipy.io.image import load_nifti
+from dipy.data import get_fnames
+"""
+For CTI analysis, data must be acquired using double diffusion encoding, taking
+into account different pairs of b-values and gradient directions between the
+two diffusion epochs, as discussed by Henriques et al. in Magn Reson Med 2021
+[NetoHe2021]._
+To run CTI we need to have separate b-value and b-vector files for each DDE
+diffusion epoch. In this tutorial, a sample DDE dataset and respective b-value
+and b-vector files for the two epochs are fetched. If you want to process your
+own DDE data compatible with CTI, you will have to change the below lines of
+code to add the paths of your data.
+The users should also ensure that the data is formatted correctly for the CTI
+analysis they are performing.
+"""
+fdata, fbvals1, fbvecs1, fbvals2, fbvecs2, fmask = get_fnames('cti_rat1')
+data, affine = load_nifti(fdata)
+bvals1, bvecs1 = read_bvals_bvecs(fbvals1, fbvecs1)
+bvals2, bvecs2 = read_bvals_bvecs(fbvals2, fbvecs2)
+"""
+In this example, the function ``load_nifti`` is used to load the CTI data from
+the file ``RB_invivo_cti_data_f3.nii`` and returns the data as a nibabel
```

Comment:
Replace "In this example, the function ``load_nifti`` is used to load the CTI data from
the file ``RB_invivo_cti_data_f3.nii`` and returns the data as a nibabel..."
by 

"In this example, the function ``load_nifti`` is used to load the CTI data saved in file``fdata`` and returns the data as a nibabel...